crates/client/src/client/schema/storage_read_replay.rs

```diff
@@ -0,0 +1,58 @@
+use super::HexString;
+use crate::client::schema::{
+    schema,
+    U32,
+};
+
+#[derive(cynic::QueryFragment, Clone, Debug)]
+#[cynic(schema_path = "./assets/schema.sdl")]
+pub struct StorageReadReplayEvent {
+    pub column: String,
+    pub key: HexString,
+    pub value: Option<HexString>,
+}
+impl From<StorageReadReplayEvent>
+    for fuel_core_types::services::executor::StorageReadReplayEvent
+{
+    fn from(event: StorageReadReplayEvent) -> Self {
+        fuel_core_types::services::executor::StorageReadReplayEvent {
+            column: event.column,
+            key: event.key.into(),
+            value: event.value.map(Into::into),
+        }
+    }
+}
+
+// mutations
+
+#[derive(cynic::QueryVariables, Debug)]
+pub struct StorageReadReplayArgs {
+    pub height: U32,
+}
+
+/// Retrieves the transaction in opaque form
+#[derive(cynic::QueryFragment, Clone, Debug)]
+#[cynic(
+    schema_path = "./assets/schema.sdl",
+    graphql_type = "Mutation",
+    variables = "StorageReadReplayArgs"
+)]
+pub struct StorageReadReplay {
+    #[arguments(height: $height)]
+    pub storage_read_replay: Vec<StorageReadReplayEvent>,
+}
+
+#[cfg(test)]
+pub mod tests {
+    use super::*;
+    use fuel_core_types::fuel_types::BlockHeight;
+
+    #[test]
+    fn execution_trace_block_tx_gql_output() {
```

Comment:
It would be good to specify that this is a compatibility regression/snapshot test. I don't get anything from this test name.

Reply:
We have loads of `*_gql_output` tests that do this. Should those all be renamed?

Reply:
> We have loads of *_gql_output tests that do this. Should those all be renamed?

Not your responsibility. I don't think upholding the precedence gains us anything here. I'd prefer clear names wherever possible.